packages/babel-plugin-proposal-do-expressions/test/fixtures/do-expressions/variable-declaration-start.js

```diff
@@ -1,4 +1,7 @@
-expect(do {
-  var bar = "foo";
-  bar;
-}).toBe("foo");
+expect(
+  do {
+    var bar = "foo";
+    bar;
+  }
+).toBe("foo");
```

Comment:
@sosukesuzuki I don't like what Prettier is doing here :stuck_out_tongue:

Reply:
What formatting is expected?

Reply:
I would have expected it to be
```js
expect(do {
  var bar = "foo";
  bar;
}).toBe("foo");
```
which is what it does with functions: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHOYYAoAmEABMADpRFEBuAhgE5EBG9RAvEaSAGYQScDc5SszqCoAXwCUAOhgQAQnFycefEJLHl0WHLlyS2APhJDqLEWw7deA0yLFTZCpSpvr+IADQgIGGAEtoAGdkUHo6CAB3AAV6BBCUGgAbSJoATxDvRjoaMABrOBgAZQxc-ygAc2QYOgBXOG84AFtGOHx8NoAZGkramgq4ADEIOiaaGADK5BAaWrkvEAALGCakgHVF-3gg0rA4Ivit-yottOmwIMyQcqC4OhhonIqx5C5k2+8AKyC0eRz8wpFGhNOCdcpwV7vBogb5oIrlCpJOAARVqEHgkKSHxApTotzo02YrSSCwwdHKMDW-nwMEWyAAHAAGbxkiC3NY5DDTMlwfFUCHeACOaPgj18CRmQQAtFA4G02gs6HBhf4lY9+i8kG8sdDbk1-NU6rqEUjUeiIVqod4YDRGFSaXSkAAma05fxJBEAYQgTU1qCCAFYFrVbgAVW0JbXYqj1ACSUA6sCKYHJfgAggmijA0kjMbdxOIgA